artifacts

```diff
@@ -39,8 +39,64 @@ prolong() {
     set_output 'link' "${URL}/builds/${ARTIFACT_TARGET}"
 }
 
+# --- cmd 1
+#    echo %(prop:eve_promotion_artifacts)s | grep -qE '^[^/]+\:(dev\-|preprod\-|)((staging|prolonged)\-[0-9]+(\.[0-9]+){1,3}\.r[0-9]{12}\.[0-9a-f]+\.[^/]+\.[0-9]+|promoted\-[^/\-]+)$';
+# --- cmd 2
+#    git tag -l | grep -E "^%%(prop:eve_promotion_artifacts)s$" | grep -qE '^[^/]+$';
+# --- cmd 3
+#    echo %(prop:eve_promotion_artifacts)s | grep -qE '^[^/]+\:(dev\-|preprod\-|)(staging|prolonged)\-[0-9]+(\.[0-9]+){1,3}\.r[0-9]{12}\.[0-9a-f]+\.[^/]+\.[0-9]+$' TODO wtf ?
+#    && artifacts_commit=`echo ${NAME} | sed -E 's/^[^/]+\:(dev\-|preprod\-|)(staging|prolonged)\-[0-9]+(\.[0-9]+){1,3}\.r[0-9]{12}\.([0-9a-f]+)\.[^/]+\.[0-9]+$/\\4/'`;
+#    echo %(prop:eve_promotion_artifacts)s | grep -qE '^[^/]+\:(dev\-|preprod\-|)promoted\-[^/]+$'
+#    && artifacts_commit=`git rev-list -n 1 \`echo %(prop:eve_promotion_artifacts)s | sed -E 's/^[^/]+\:(dev\-|preprod\-|)promoted\-([^/]+)$/\\2/')\``;
+#    tag_commit=`git rev-list -n 1 %(prop:eve_promotion_tag)s`;
+#    case ${tag_commit} in
+#      ${artifacts_commit}*) exit 0;;
+#      *) exit 1;;
+#    esac;
+# --- cmd 4
+#     echo %(prop:eve_promotion_artifacts)s | grep -qE '^[^/]+\:(dev\-|preprod\-|)(staging|prolonged)\-[0-9]+(\.[0-9]+){1,3}\.r[0-9]{12}\.[0-9a-f]+\.[^/]+\.[0-9]+$'
+#     && artifacts_prefix=`echo %(prop:eve_promotion_artifacts)s | sed -E 's/(^[^/]+\:)(dev\-|preprod\-|)(staging|prolonged)\-[0-9]+(\.[0-9]+){1,3}\.r[0-9]{12}\.[0-9a-f]+\.[^/]+\.[0-9]+$/\\1\\2/'`;
+#     echo %(prop:eve_promotion_artifacts)s | grep -qE '^[^/]+\:(dev\-|preprod\-|)promoted\-[^/]+$'
+#     && artifacts_prefix=`echo %(prop:eve_promotion_artifacts)s | sed -E 's/(^[^/]+\:)(dev\-|preprod\-|)promoted\-[^/]+$/\\1\\2/'`;
+#     artifacts_target=${artifacts_prefix}promoted-%(prop:eve_promotion_tag)s;
+#     echo "copying '%(prop:eve_promotion_artifacts)s' to '${artifacts_target}'";
+#     curl --silent http://artifacts/copy/%(prop:eve_promotion_artifacts)s/${artifacts_target}/ | tee /tmp/%(prop:workername)s-promotion-output.log;
+# --- cmd 5
+#    tail -1 /tmp/%(prop:workername)s-promotion-output.log | grep -qE '^BUILD COPIED$';
```

Comment:
I know you know this, but just writing it down so that we don't forget. Let's remove those lines